platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -499,12 +499,23 @@ namespace platoon_strategic_ihp
 
                 auto llts = wm_->getLaneletsFromPoint(current_loc, 1);
 
-                auto geofence_gaps = llts[0].regulatoryElementsAs<lanelet::DigitalMinimumGap>();
                 double lanelet_digitalgap = 0.0;
```

Comment:
Seems like a default value should be used to make this safe? We wouldn't want platooning to control to 0 gap